### PR TITLE
feat: implement simple UI for the Image Service

### LINKED ISSUE #18: HTMX frontend

### PR DESCRIPTION
**Pull request by smira**
Fixes #18

The intention is to provide educational frontend so that people can get comfortable using the Image Service by building the appropriate links.

### CODE REVIEW

**Review by Unix4ever — APPROVED** [review]

File: .kres.yaml [hunk 1]
```diff
@@ -113,4 +113,9 @@ spec:
 ---
 kind: auto.CI
 spec:
-    provider: drone
+  provider: drone
```
Comment (frezbo) [inline .kres.yaml:116]: this can already use github actions

File: internal/frontend/http/ui.go [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+
+	"github.com/siderolabs/image-service/internal/artifacts"
+	"github.com/siderolabs/image-service/pkg/flavor"
+)
+
+//go:embed css/output.css
+var cssFS embed.FS
+
+//go:embed js/*
+var jsFS embed.FS
```
Comment (frezbo) [inline internal/frontend/http/ui.go:29]: oh this is cool, didn't know we could do this

**Review by frezbo — APPROVED** [review]

File: .kres.yaml (hunk 1 above)
Comment (smira) [reply .kres.yaml:116]: one step at a time, I need to move to GHA, but in a separate PR :)

**Pull request merged**